### DISCUSSION TITLE
Need Pagination for Posts

### DISCUSSION THREAD

**Discussion by jmalinie** [GetPublii/Publii#1440] [General]
İ have about 2000 posts and publii begin to struggle  loading the posts, i want to have pagination for posts eg 100 posts or 50 exct per page so it can load the only what is needed.  it i,s curicial for my business, as i m planning to load 30 posts every 5 minutes, after adding about 20-30 posts publii gives white screen, it is it is using much more Ram, and at the end ram is finishing.  Can publii team or developers create this function ?

Free or paid is oke for me . 

Thanks

**Comment by dziudek** [comment]
Hi,

Virtual scrolling for post/tag lists is planned for v.046-0.47 release so it probably will solve issue with rendering bigger lists of posts

**Reply by jmalinie** [reply]
Ho great, ! Really thank you.  Will help me alot . However, is new update release date planned ? Please add this function in new release.

Thanks

**Reply by dziudek** [reply]
@jmalinie - next release (v.0.46) is planned for second half of April '24, but if you need I can send you prerelease version with this feature as soon as it will be ready. Please only let me know which platform are you using :) Then I will note to let you know as soon as this feature will be available

**Reply by jmalinie** [reply]
@dziudek  This would be great. i really need in urgent to this feature. i use Windows 11 Home. 

Thanks for your kind help.

**Reply by dziudek** [reply]
Ok, I will do my best to provide this feature as soon as possible :)

**Comment by jmalinie** [comment]
> Ok, I will do my best to provide this feature as soon as possible :)

Hi, 
if you can provide me updated version, i will be glad. 

Thanks

**Reply by dziudek** [reply]
@jmalinie - unfortunately the problem is much deeper and bigger than I initially thought. At this moment I still have no solution. But it is not strictly related to the rendering, but to the post filters. I will come back to this topic when I will have more time
